libraries/stdlib/src/kotlin/jvm/JvmFlagAnnotations.kt

```diff
@@ -19,18 +19,39 @@ package kotlin.jvm
 import java.lang.annotation.Retention
 import java.lang.annotation.RetentionPolicy
 
+/**
+ * Marks the JVM backing field of the annotated property as `volatile`, meaning that it can
+ * be modified asynchronously by concurrently running threads.
```

Comment:
"can be modified async" is strange wording. Even if field is not volatile, you still can modify it async by concurrent threads, but effect may be different. Need better description.